### PR TITLE
Add titles on profile icons (contextual help on hover)

### PR DESCRIPTION
**Pull request by loicgasser**
@davidoesch 

I had prepared that some time ago when I realized I couldn't tell what was what.
From left to right we need the following translations.

1. profile_elevation_difference
2. profile_elevation_up
3. profile_elevation_down
4. profile_poi_up
5. profile_poi_down
6. profile_distance
7. profile_slope_distance
8. profile_hike_time

ping @oterral for the review

[TestLink](https://mf-geoadmin3.int.bgdi.ch/gal_profile_tooltip/index.html)

### CODE REVIEW

**Comment by davidoesch** [comment]
It's all here in all languages http://help.geo.admin.ch/?id=94&lang=de

**Comment by loicgasser** [comment]
Nice thanks

**Comment by loicgasser** [comment]
translations added

File: src/components/profile/style/profile.less [hunk 1]
```diff
@@ -54,23 +54,27 @@
...
-  
+
   .ga-profile-legend,
   .ga-profile-label {
     font-weight: bold;
     text-shadow: 1px 1px lightgrey;
   }
 
+  text {
+    cursor: default;
```
Comment (oterral) [inline src/components/profile/style/profile.less]: Why not leaving the auto cursor . It showed that you can select then copy the text

Comment (loicgasser) [inline src/components/profile/style/profile.less]: I thought it made more sense with the tooltip. Not sure people want to copy paste that. But I am open to suggestions.

Comment (oterral) [inline src/components/profile/style/profile.less]: Ok I can live with that.

**Pull request merged**